jax/interpreters/pxla.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2019 Google LLC
```

Comment:
I don't have a preference either way, but I've seen people say recently that copyright dates shouldn't be updated since they're a marker of first creation not, like, the current date. I trust your judgement on what our convention should be!

Reply:
I agree with the convention, but the intent of the change here was: the initial copyright date was wrong. This file (and a number of others added recently) have incorrectly stated a date of 2018 even though it's now 2019.

Reply:
Oh, you're right! Thanks for catching that.